.gitignore

```diff
@@ -11,3 +11,56 @@ _notebooks/.ipynb_checkpoints
 .netlify
 .tweet-cache
 __pycache__
+_posts/2020-03-11-Mortality_Rate.md
+_posts/2020-03-12-covid19-us-states.md
+_posts/2020-03-13-COVID19-Comparitive-Analysis.md
+_posts/2020-03-13-EDA.md
+_posts/2020-03-14-covid19_growth.md
+_posts/2020-03-15-us-growth-by-state-map.md
+_posts/2020-03-16-covid19_growth_bayes.md
+_posts/2020-03-17-covid19-compare-country-death-trajectories.md
+_posts/2020-03-17-covid19-compare-country-trajectories.md
+_posts/2020-03-18-case-count-estimation-us-states.md
+_posts/2020-03-19-cases-and-deaths-per-million.md
+_posts/2020-03-19-estimating_infected.md
+_posts/2020-03-21-covid19-overview-europe.md
+_posts/2020-03-21-covid19-overview-us.md
+_posts/2020-03-21-covid19-overview.md
+images/copied_from_nb/covid-compare-1-1.png
```

Comment:
Why are you ignoring this?  This will break the build system as images need to be saved